qemu/tests/cfg/multi_disk.cfg

```diff
@@ -5,7 +5,18 @@
     remove_image = yes
     remove_image_image1 = no
     cmd_timeout = 1000
-    black_list = C:
```

Comment:
Since the 'Linux:' part has its specific cmd 'rmpart_cmd = 'parted -s %s rm 1'', so even move 'list_disk_cmd = 'lsblk -oKNAME,MOUNTPOINT,SIZE,MODEL'' out of the part, it won't reduce the code quantity. Thus I just keep the original code layout which is more clear to understand.